ui/src/state/ephemeral.ts

```diff
@@ -0,0 +1,21 @@
+import create from 'zustand';
```

Comment:
we use "local.ts" for random stuff like this

Reply:
was going to add there, but looked like that still gets persisted to `localStorage`?

this felt like something that should fully reset when you close the app

Reply:
jk, missed the partialize call there. Will update